pkgs/applications/networking/cluster/helmsman/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildGoModule, fetchFromGitHub, ... }:
+
+let version = "3.3.0"; in
+
+buildGoModule {
+  pname = "helmsman";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "Praqma";
+    repo = "helmsman";
+    rev = "v${version}";
+    sha256 = "0i7sg3iwxb07gjxcz6chpdcx3fqykzldmf7s1c9m02hkps910ca8";
+  };
+
+  goPackagePath = "github.com/Praqma/helmsman";
```

Comment:
`goPackagePath` is required for legacy `buildGoPackage`, but no longer for `buildGoModule`.